roles/network_plugin/calico/tasks/check.yml

```diff
@@ -44,19 +44,11 @@
   delegate_to: "{{ groups['kube_control_plane'][0] }}"
 
 - name: Get current calico cluster version
```

Comment:
change name to: `Get current calico version`

Reply:
Thanks for your review.
I'll update it in new commit